central/pruning/pruning_test.go

```diff
@@ -567,7 +567,7 @@ func (s *PruningTestSuite) TestImagePruning() {
 			alerts, config, images, deployments, pods, indexQ := s.generateImageDataStructures(ctx)
```

Comment:
Need to add a test for PLOP pruning.

Reply:
We have added three tests for PLOP pruning.

Test that open PLOP is not pruned.
Test that closed PLOP which is not expired is not pruned.
Test that closed PLOP which is expired is pruned.